packages/manager/src/features/IAM/Users/UsersTable/CreateUserDrawer.tsx

```diff
@@ -113,25 +95,6 @@ export const CreateUserDrawer = (props: Props) => {
           rules={{ required: 'Email is required' }}
         />
 
-        <Controller
-          render={({ field }) => (
-            <FormControlLabel
-              label={`This user will have ${
-                field.value ? 'limited' : 'full'
-              } access to account features.
-                    This can be changed later.`}
-              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                field.onChange(!e.target.checked);
-              }}
-              checked={!field.value}
-              control={<Toggle data-qa-create-restricted />}
-              sx={{ marginTop: 1 }}
-            />
-          )}
-          control={control}
-          name="restricted"
-        />
-
         <Box sx={{ marginTop: 1 }}>
```

Comment:
```suggestion
        <Box sx={{ marginTop: 2 }}>
```
Nit: Spacing looks a little off here now, with the notice being so close to the bottom of the textfield, and 16px top margin might be more balanced.

| Prod | This Branch | Suggested |
|--------|--------|--------|
| ![Screenshot 2025-03-18 at 8 38 02 AM](https://github.com/user-attachments/assets/3e35c796-a5bb-4609-9a32-7336f751a263) | ![Screenshot 2025-03-18 at 8 37 19 AM](https://github.com/user-attachments/assets/3f6e274f-4e56-4e98-830a-a6a3ed2fbe0a) | ![Screenshot 2025-03-18 at 8 44 55 AM](https://github.com/user-attachments/assets/8dbad330-6da0-42e5-ab22-1a42b669e5ff) |

Reply:
sure, makes sense, done